src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -3556,9 +3556,13 @@ public sealed partial class Version : System.IComparable, System.IComparable<Sys
         public static bool operator <(System.Version v1, System.Version v2) { throw null; }
         public static bool operator <=(System.Version v1, System.Version v2) { throw null; }
         public static System.Version Parse(string input) { throw null; }
+        public static System.Version Parse(ReadOnlySpan<char> input) { throw null; }
         public override string ToString() { throw null; }
         public string ToString(int fieldCount) { throw null; }
+        public bool TryFormat(System.Span<char> destination, out int charsWritten) { throw null; }
```

Comment:
Ideally we wouldn't fork between netcoreapp and uap, but I don't know the current state of Span support in uap. @stephentoub probably knows more then I do on that front.